docs/data-guidelines.md

```diff
@@ -280,3 +281,26 @@ Members of this mixin are available to `HTMLAnchorElement` and `HTMLAreaElement`
    ```
 
 This guideline was proposed in [#8929](https://github.com/mdn/browser-compat-data/issues/8929), based in part on previous discussion in [#472](https://github.com/mdn/browser-compat-data/issues/472).
+
+## APIs moved on the prototype chain
+
+[Web IDL interfaces](https://heycam.github.io/webidl/#idl-interface) (and [JavaScript built-in objects](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects)) form prototype chains, with one type inheriting from another. For example, `AudioContext` inherits from `BaseAudioContext`, and `Element` inherits from `Node`.
+
+Some of these interfaces are [abstract](https://en.wikipedia.org/wiki/Abstract_type) and never have instances, while most are concrete and can be instantiated. For example, `BaseAudioContext` and `Node` are abstract, while `AudioContext` and `Element` are concrete.
+
+When attributes and methods are moved between interfaces in specifications and implementations, BCD should make the corresponding change. This guidelines covers what versions to use, and whether to use `partial_implementation` and notes in the resulting compat data.
+
+**When members are moved up the prototype chain**, such as from `AudioContext` to `BaseAudioContext`:
+
+- For the members, use the versions when they were first supported on any concrete interface inheriting from `BaseAudioContext`, even if that is earlier than `BaseAudioContext` itself existed. Use `partial_implementation` and notes for the members if there are concrete interfaces where the member wasn't supported prior to the move.
+- For the interface itself, add a `partial_implementation` entry going back as far as the earliest member, and a simple entry from the time the interface itself was introduced.
+
+The [`SVGGeometryElement`](https://github.com/mdn/browser-compat-data/pull/9479) move is an example of this.
+
```

Comment:
Can we rework this section to declare a general principle then illustrate it with an example? I'm 95% sure I know what you're getting at here, but without a statement of the principle, I'm not sure.

Something along these lines:

> ### When  members are moved up the prototype chain
> For the members, use the versions when the member was a part of any concrete interface in the prototype chain, even if it was earlier than the existence of the current interface itself. For the interface, use `partial_implementation` and a note on the interface if there are concrete interfaces where the member was not supported before the move.
>
> For example, for member `x` moved from `AudioContext` to `BaseAudioContext`. The data was recorded like this:
> - `x` was removed from `AudioContext`
> - `x` was added to `BaseAudioContext`, with support from the first version supported on `AudioContext`
> - `BaseAudioContext` was shown as having `partial_implementation` from…
> - etc. etc. etc.